rllib/examples/env/halfcheetah_rand_direc.py

```diff
@@ -1,6 +1,15 @@
 import numpy as np
 import gym
 from gym.envs.mujoco.mujoco_env import MujocoEnv
+"""
```

Comment:
Can you actually make an interface for this? Something like:

rllib/envs/meta_learning_env.py
```
class MetaLearningEnv:
    def set_task(self, task_id: int) -> None
       raise NotImplemented
    def sample_tasks(self, num_tasks: int): -> List[int]
       raise NotImplemented
```

Then it can be documented in one place, and these envs can inherit from this to make the interface requirement clear (use `@override(MetaLearningEnv)` for the set task sample task methods).